### ISSUE TITLE
datetime object dayofyear and dayofwk issue

### ISSUE THREAD

**Issue by djkirkham** [Unidata/cftime#13]
`netcdftime.datetime` objects constructed directly don't have the correct day of year or day of week; they always give a value of 1 and -1 respectively. Objects returned by `utime.num2date` have the correct values, except for those with Julian calendars, which still have the issue.

```
>>> import netcdftime
>>> print netcdftime.__version__
1.4.1
>>> date = netcdftime.DatetimeNoLeap(2000, 1, 2)
>>> print repr(date)
netcdftime._netcdftime.DatetimeNoLeap(2000, 1, 2, 0, 0, 0, 0, -1, 1)
>>> print date.dayofyr 
1
>>> print date.dayofwk
-1
```
Python version: 2.7
netdf4 version: 1.2.7
Installed via conda forge.

**Comment by jswhit** [comment]
The dayofwk and dayofyr values only get calculated when the routine is called to convert a Julian day number to a date (i.e. num2date).  The only way that cftime.DatetimeNoLeap will have the correct values is if they are passed into __init__.  This may be a flaw in the design, but that's the way it is for now.

**Comment by jswhit** [comment]
After looking into this a bit more, it seems to me that the dayofwk and dayofyr support is half-baked at best.  These attributes are not provided by the python datetime object, so perhaps we should just remove them.  I wonder if anyone is actually using this?

**Comment by jswhit** [comment]
This is fixed by pull request #66, closingnow.
